Add new person-vehicle-bike detection models

GalyaZalesskaya opened the pull request:
Added person-vehicle-bike-detection-2003 and person-vehicle-bike-detection-2004 models as a release candidates for OpenVINO 2021.3.

Anna-Mironova commented on models/intel/person-vehicle-bike-detection-2004/description/person-vehicle-bike-detection-2004.md:
```diff
@@ -0,0 +1,51 @@
+
+## Inputs
+
+Name: `input`, shape: [1x3x256x448] - An input image in the format [BxCxHxW],
```
```suggestion
Name: `input`, shape: `1, 3, 256, 448` - An input image in the format `B, C, H, W`,
```

Anna-Mironova commented on models/intel/person-vehicle-bike-detection-2004/description/person-vehicle-bike-detection-2004.md:
```diff
@@ -0,0 +1,51 @@
+
+## Outputs
+
+1. The `boxes` is a blob with shape: [N, 5], where N is the number of detected
```
```suggestion
1. The `boxes` is a blob with shape: `N, 5`, where N is the number of detected
```

Anna-Mironova commented on models/intel/person-vehicle-bike-detection-2004/description/person-vehicle-bike-detection-2004.md:
```diff
@@ -0,0 +1,51 @@
+    - (`x_min`, `y_min`) - coordinates of the top left bounding box corner
+    - (`x_max`, `y_max`) - coordinates of the bottom right bounding box corner.
+    - `conf` - confidence for the predicted class
+2. The `labels` is a blob with shape: [N], where N is the number of detected
```
```suggestion
2. The `labels` is a blob with shape: `N`, where N is the number of detected
```

eaidova commented:
Jenkins please retry a build

vladimir-dudnik commented:
if new models should work with old demos, please add models to relevant demo's models.lst and create test cases for them (but leave them commented out). We will uncomment when new models will be available to downloader.

GalyaZalesskaya commented:
@eaidova Thank you for your patience, I had to do a huge investigation, but I fixed the drop.
@vladimir-dudnik I added unit test case and commented it. Also, I checked that the rule to pick relevant models in model.lst already covers new models, so there are no changes.

vladimir-dudnik commented:
please see pre-commit checks failed

13:12:07  ./models/intel/person-vehicle-bike-detection-2004/accuracy-check.yml
13:12:07    19:14     error    too many spaces after hyphen  (hyphens)
13:12:07  
13:12:07  ./models/intel/person-vehicle-bike-detection-2003/accuracy-check.yml
13:12:07    19:14     error    too many spaces after hyphen  (hyphens)

eaidova commented on models/intel/person-vehicle-bike-detection-2003/accuracy-check.yml:
```diff
@@ -0,0 +1,26 @@
+
+    launchers:
+      - framework: dlsdk
+        device: CPU
```
please remove device

eaidova commented:
Jenkins please retry a build

xonobo commented:
Cannot download 2003 and 2004 models. 
`python downloader.py --print_all`
command does not list these models.
